tap/plugin.py

```diff
@@ -55,4 +61,6 @@ def _cls_name(self, test):
         return test.test.__class__.__name__
 
     def _description(self, test):
+        if self._format:
+            return self._format.format(sd=test.shortDescription(), mn=str(test))
```

Comment:
Any format string that does not match the naming will raise a `KeyError` here. Can this code catch the `KeyError` and exit with something like `sys.exit('Bad format string: {0}\nReplacement options are: \{short_description\} and \{method_name\}')`?